### PR TITLE
Update tax provider API documentation to specify expectations for tax exempt items in equivalent response item.

### PR DESCRIPTION
**Pull request by theromulans**
# [TAX-2052]

## What changed?
* More detail around our enforced expectations for tax exempt items.

[TAX-2052]: https://bigcommercecloud.atlassian.net/browse/TAX-2052?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jvpcode — APPROVED** [review]

**Review by benpratt77 — APPROVED** [review]

File: reference/tax_provider.yml [hunk 1]
```diff
@@ -923,7 +923,7 @@ components:
           $ref: '#/components/schemas/TaxClass'
         tax_exempt:
           type: boolean
-          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. Tax-exempt items are included in the request for auditing purposes.'
+          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. Tax-exempt items are included in the request for auditing purposes. Tax-exempt items must have a tax amount of zero within the tax quote response.'
```
Comment (bc-traciporter) [inline reference/tax_provider.yml:926]:
```suggestion
          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. The request body consists of tax-exempt items for auditing purposes. Tax-exempt items must have a tax amount of zero within the tax quote response.'
```
Removed passive voice

**Review by bc-traciporter — COMMENTED** [review]
Removed passive voice

File: reference/tax_provider.yml (hunk 1 above)
Comment (theromulans) [reply reference/tax_provider.yml:926]: Thanks @bc-traciporter!

**Review by agchan12 — APPROVED** [review]

File: reference/tax_provider.yml (hunk 1 above)
Comment (bc-traciporter) [reply reference/tax_provider.yml:926]: Ok, no problem

**Review by bc-traciporter — APPROVED** [review]

**Review by bc-andreadao — APPROVED** [review]

**Pull request merged**